python/pyproject.toml

```diff
@@ -28,6 +28,7 @@ langsmith = "langsmith.cli.main:main"
 python = ">=3.8.1,<4.0"
 pydantic = ">=1,<3"
 requests = "^2"
+pytest-subtests = "^0.11.0"
```

Comment:
This lets us run subtests

Reply:
could this be separated to a something like [tool.poetry.group.test.dependencies] so it doesn't install for prod builds?

Reply:
Yeah that's my bad, let me put up a fix

Reply:
https://github.com/langchain-ai/langsmith-sdk/pull/293